src/api/entities/SecurityToken/Permissions.ts

```diff
@@ -52,6 +59,11 @@ export class Permissions extends Namespace<SecurityToken> {
    */
   public inviteAgent: ProcedureMethod<InviteExternalAgentParams, void>;
 
+  /**
+   * Remove an external agent from this Security Token
```

Comment:
"Revoke an Agent's permissions for this Security Token"

Also, please change the comment for `inviteAgent` to "Invite an Identity to be an Agent with permissions over this Security Token"

Change the comment in `createGroup` to "Create a Permission group for this Security Token. Identities can then be assigned to said group as Agents. Agents assigned to a group have said group's permissions over this Security Token"